scripts/run-gtest.ps1

```diff
@@ -138,6 +157,13 @@ $TestExeName = Split-Path $Path -Leaf
 $LogDir = Join-Path $RootDir "artifacts" "logs" $TestExeName (Get-Date -UFormat "%m.%d.%Y.%T").Replace(':','.')
 New-Item -Path $LogDir -ItemType Directory -Force | Out-Null
 
+# Folder for coverage files
+$CoverageDir = $null
+if ($CodeCoverage) {
+    $CoverageDir = Join-Path $RootDir "artifacts" "coverage"
+    New-Item -Path $CoverageDir -ItemType Directory -Force | Out-Null
+}
```

Comment:
We need to support running in isolated mode, where each test is a different execution of the test binary. Then merge all executions back up. It might be as simple as using the log directory as the temporary out and then merge it into this path after each test.